### ISSUE TITLE
Root neded

### ISSUE THREAD

**Issue by ELJOKER900** [TermuxHackz/wifi-hacker#29]
I dont hace root and i cant root my phone and i cant execute it

**Comment by TermuxHackz** [comment]
Need to be rooted

**Comment by Loqueseamevaleverg** [comment]
Yup, root it first. Make sure your bootloader is unlocked. There are
non-unlocking rootings, but it depends on the model.
